pom.xml

```diff
@@ -592,8 +592,7 @@
                                     <goal>verify</goal>
                                 </goals>
                                 <configuration>
-                                    <!-- Sets the VM argument line used when integration tests are run. -->
-                                    <argLine>${argLine}</argLine>
+                                    <argLine>-Dio.netty.leakDetection.level=advanced</argLine>
```

Comment:
The previous line was mainly used to get jacoco working correctly to report test coverage and I don't think we need it for stability tests.

>  Is it possible to define the value in the environment so that we don't need to change the code?

Do you mean configure `leakDetection.level` using environment variables? Nope, I tried and it doesn't work.

Reply:
> Do you mean configure leakDetection.level using environment variables?

Yep